### DISCUSSION TITLE
Volumetric Flow Rate - Shouldn't it be on Printer Settings - Extruder Page?

### DISCUSSION THREAD

**Discussion by Sandoz25** [supermerill/SuperSlicer#2819] [Community contribution]
While, I can certainly see the use case of having a Volumetric Flow Rate limited in the Print Settings but I feel there should be one on the Extruder Tab, in the printer settings but I feel in the print settings it should be the desired or requested rate and in the printer settings it should be a limit. 

I think one of the great things with Super Slicer is the ability to use the same Print Settings profile with different printer settings to be able to print the same part on multiple machines with relatively similar results.  If I have several machines with different hotends (E3d v6, revo, magnum, etc) but are the same otherwise, I want to be able to use the same profiles, and the speeds be limited by either that machines limist of speed and acceleration or the flow rate of that machine.  

By having it on the print profile, I either have to make a seperate print profile for each machine (bad, if you ever want to change your settings) or not use the volumetric flow rate setting, or set it to the speed of your slowest printer and nor be able to use the full potential of the higher flow hot ends.

It is possible I'm completely clueless and have overlooked this setting or it works in a way other than it appears to.  To which I appologize for wasting your time.  But if you could point me to it, then not all would be lost.

Banana for scale

⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠛⠋⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⣿⡇⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣸⣿⣿⣶⡀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣿⣿⡛⢻⣷⡀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣾⣿⣿⠃⢸⣿⡇⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣾⣿⣿⡟⢀⣿⣿⠃⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣿⣿⣿⡿⠁⣼⣿⡏⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⣿⣿⠟⢀⣾⣿⡟⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣴⣾⣿⣿⣿⠟⢁⣴⣿⣿⠋⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣴⣾⣿⣿⣿⣿⠿⠋⣁⣴⣿⣿⠟⠁⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⢀⣠⣴⣾⣿⣿⣿⣿⡿⠟⠋⣁⣴⣾⣿⡿⠟⠁⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⢠⣾⣿⣿⣿⣿⠿⠟⠋⣁⣤⣶⣿⣿⡿⠟⠋⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠈⠉⢉⣁⣤⣴⣶⣾⣿⣿⠿⠟⠋⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠉⠉⠛⠉⠉⠉⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀

**Comment by neophyl** [comment]
Volumetric flow rate is set in 2 places.  Its on the Printer profile and also the filament profile.  The lower value of the 2 is used normally.
Different filaments have different max flow rates with the same extruder setup which is why there is one on the filaments profile, its not just a measure of what your extruder is capable of.

I agree that the setting is probably better on the Printer profile though.  However given its upstream from PS is also there and where its been forever even with the original Slic3r then changing it has the potential to break a lot of existing profiles.  Probably the main reason Prusa doesnt touch it.

I'd like to see a 3rd one on the Extruder page, call it Extruder Max Volumetric Flow to differentiate it and then it use the lowest of all 3.  That way you could disable the one on the print profile (or set it way high so its not a factor if you didnt want to use it) and then limit it with the Extruder one and then the filament one could modify if needed for material choice.

**Comment by supermerill** [comment]
#491

#2452
